features/source.yml

```
@@ -0,0 +1,19 @@
+name: <source>
+description: The `<source>` element specifies a media resource for the `<picture>`, `<video>`, or `<audio>` elements.
```

I like this, but there's a couple of style issues.

First, see [the guidance on _specifies_](https://github.com/web-platform-dx/web-features/blob/main/docs/README.md#specifies.

The current phrasing with "or" also sound ambiguous to me, as if it's not known which of the elements it applies to. One of these alternatives might do.

```suggestion
description: The `<source>` element sets a media resource for an `<picture>`, `<video>`, or `<audio>` element.
```

```suggestion
description: The `<source>` element sets a media resource for the `<picture>`, `<video>`, and `<audio>` elements.
```

---

done.